test/sde/sde_additive_tests.jl

```diff
@@ -60,6 +60,6 @@ sim2 = test_convergence(dts,prob,SOSRA2(),numMonte=5)
 dts = (1/2) .^ (11:-1:7) #14->7 good plot
 Random.seed!(100)
 sim2 = test_convergence(dts,prob,SKenCarp(),numMonte=20)
-@test abs(sim2.𝒪est[:l∞]-2) <.1 #High tolerance since low dts for testing!
-sim2 = test_convergence(dts,prob,SKenCarp(nlsolve=StochasticDiffEq.NLNewton(min_iter=3)),numMonte=20)
-@test abs(sim2.𝒪est[:l∞]-2) <.1 #High tolerance since low dts for testing!
+@test_broken abs(sim2.𝒪est[:l∞]-2) <.1 #High tolerance since low dts for testing!
+sim2 = test_convergence(dts,prob,SKenCarp(nlsolve=StochasticDiffEq.NLNewton()),numMonte=20)
+@test_broken abs(sim2.𝒪est[:l∞]-2) <.1 #High tolerance since low dts for testing!
```

Comment:
What if you bump `numMonte` a bit? Would that make tests pass?

Reply:
I tried just now and even increasing upto 100 wont make it pass.

Reply:
What is it converging like? 1.5?

Reply:
Yes, around 1.4

Reply:
Change it to 1.5. Looks like that conjecture is false.

Reply:
Ohh, on it. Also sometimes the convergence tests above would also fail at random times. This would fix them too I guess.